commands-v2/drive-distance-offboard/commands/drivedistanceprofiled.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+import commands2
+
+import wpimath.trajectory
+
+import subsystems.drivesubsystem
+import constants
+
+
+class DriveDistanceProfiled(commands2.TrapezoidProfileCommand):
+    """Drives a set distance using a motion profile."""
+
+    def __init__(
+        self, meters: float, drive: subsystems.drivesubsystem.DriveSubsystem
+    ) -> None:
+        """
+        Creates a new DriveDistanceProfiled command.
+
+        :param: meters The distance to drive.
+        :param: drive The drive subsystem to use.
```

Comment:
Please use our Javadoc converter as outlined in https://github.com/robotpy/examples/blob/main/CONTRIBUTING.md#include-all-the-comments